java/main/com/google/privacy/differentialprivacy/GaussianNoise.java

```diff
@@ -114,6 +114,32 @@ public MechanismType getMechanismType() {
     return MechanismType.GAUSSIAN;
   }
 
+  /**
+   * Returns {@link ConfidenceInterval} object with the given {@code confidenceLevel},
```

Comment:
nit: specifying what method returns in a doc is not very helful since this is clear from the method signature any way.

I would phrase this as "Computes a confidence interval that contains the raw value {@code x} passed to {@link #addNoise(double, int, double, double, Double)} with a probability greater or equal to {@code confidenceLevel} based on the specified {@noisedValue} and noise parameters."